content/cpp/concepts/smart-pointers/smart-pointers.md

```diff
@@ -0,0 +1,82 @@
+---
+Title: 'Smart Pointers'
+Description: 'A smart pointer is a class that holds and manages a pointer through RAII.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Smart Pointers'
+  - 'Pointers'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+## Smart Pointers
```

Comment:
Delete this line right here because actually don't need this.

Reply:
`## Smart Pointers`

Reply:
Hope you don't mind @yekyam, I removed the heading. If there's any confusion: for each entry, the introductory line/paragraph does not include a header.